yarn.lock

```diff
@@ -19338,9 +19333,11 @@ p-is-promise@^2.0.0:
   integrity sha512-Y3W0wlRPK8ZMRbNq97l4M5otioeA5lm1z7bkNkxCka8HSPjR0xRWmpCmc9utiaLP9Jb1eD8BgeIxTW4AIF45Pg==
 
 p-limit@^1.1.0:
-  version "1.1.0"
-  resolved "https://registry.yarnpkg.com/p-limit/-/p-limit-1.1.0.tgz#b07ff2d9a5d88bec806035895a2bab66a27988bc"
-  integrity sha1-sH/y2aXYi+yAYDWJWiurZqJ5iLw=
+  version "1.3.0"
```

Comment:
This change doesn't seem related to browserlist or caniuse-lite.

Accidental dedupe?

Reply:
That's odd, I made sure all dedupes were for specific packages. Let me fix that.

Reply:
Force pushed an updated `yarn.lock`. please take another look!